### PR TITLE
Update UserInterfaceSettingsActivity.kt

### PR DESCRIPTION
**Pull request by sneazy-ibo**
## Restart prompt for home layout changes
- added components require restart
- removed components require reload

→ prompt shows after changes in the home layout are done

### CODE REVIEW

**Comment by Sadwhy** [comment]
I'm guessing it just adds a ok button that restarts the app

**Comment by sneazy-ibo** [comment]
> I'm guessing it just adds a ok button that restarts the app

The ok button is from the Show/Hide Layouts on Home popup and not from the reload prompt

**Comment by Sadwhy** [comment]
I completely forgot that `restartApp()` was a custom function and not something from android. Just use "Done" or "Confirm" isn't of ok. Looks weird.

**Comment by Sadwhy** [comment]
Change and reload? :trollface:

File: app/src/main/java/ani/dantotsu/settings/UserInterfaceSettingsActivity.kt [hunk 1]
```diff
@@ -52,6 +52,25 @@ class UserInterfaceSettingsActivity : AppCompatActivity() {
                 }.show()
```
Comment (rebelonion) [inline app/src/main/java/ani/dantotsu/settings/UserInterfaceSettingsActivity.kt:52]: you didn't delete the old function

**Review by rebelonion — CHANGES_REQUESTED** [review]

**Review by aayush2622 — APPROVED** [review]
all good

**Pull request merged**